src/wallet/wallet.cpp

```diff
@@ -2036,8 +2036,6 @@ OutputType CWallet::TransactionChangeType(const std::optional<OutputType>& chang
             any_wpkh = true;
         } else if (type == TxoutType::SCRIPTHASH) {
             any_sh = true;
-        } else if (type == TxoutType::PUBKEYHASH) {
-            any_pkh = true;
```

Comment:
You can also delete the creation of the variable `any_pkh` in 2028, since it's not used anywhere.